wire-ios/Wire-iOS/Sources/UserInterface/MainController/MainTabBarController.swift

```diff
@@ -99,3 +110,13 @@ private struct MainTabBarControllerWrapper: UIViewControllerRepresentable {
 
     func updateUIViewController(_ tabBarController: UITabBarController, context: Context) {}
 }
+
+extension UIImage {
+
+    fileprivate static func from(solidColor color: UIColor) -> UIImage {
```

Comment:
I think one can use `private` nowadays
```suggestion
    private static func from(solidColor color: UIColor) -> UIImage {
```

Reply:
Actually not, since the extension is for `UIImage` and it's used outside of the `UIImage` class.